charts/k0s/templates/syncer.yaml

```diff
@@ -18,6 +18,7 @@ metadata:
 {{ toYaml $annotations | indent 4 }}
   {{- end }}
 spec:
+  publishNotReadyAddresses: true
```

Comment:
I think this is wrong here

Reply:
I don't believe it is, it would not create a working vcluster without it because we are waiting on etcd to be available before starting the api, and etcd would not start because it would not have any record for `vcluster-headless` service (or something like that)
we have the same in k3s https://github.com/loft-sh/vcluster/blob/52324ca83eaec37c35d25cc202e2c59db29d6526/charts/k3s/templates/statefulset-service.yaml#L18 I believe for the same reason